style.css

```diff
@@ -0,0 +1,135 @@
+* {
+  padding: 0;
+  margin: 0;
+  list-style-type: none;
+  text-decoration: none;
+  box-sizing: border-box;
+  font-family: 'Poppins', sans-serif;
+}
+
+header {
+  display: flex;
+  width: 100%;
+  height: 100vh;
+  padding: 0 200px;
+}
+
+header nav {
+  display: flex;
+  justify-content: space-between;
+  padding: 20px 150px 50px 150px;
+}
+
+header nav .logo a {
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  color: #6070ff;
+}
+
+header .toggle-button {
+  display: none;
+}
+
+header nav #hamburger-menu {
+  display: flex;
+}
+
+header nav #hamburger-menu li a {
+  display: block;
+  padding: 0 1rem;
+  padding-right: 0;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #344563;
+}
+
+header .head-body-container {
+  background-image: url(image/Header-bg.png);
+  background-repeat: no-repeat;
+  background-size: 100%;
+  padding: 0 100px;
+}
```

Comment:
- According to the project requirements, the content of this section should be aligned using the `flex` display property. Please fix this 👍 
- Also, the bottom-left part of this section has a border-radius, please implement this in your design, remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏

Your project:
<img width="238" alt="wrfrfwf" src="https://user-images.githubusercontent.com/73607512/212017881-7351abcc-40fa-4e3e-8f7e-fc9ba741bb94.PNG">

Figma design:
<img width="144" alt="egnengn" src="https://user-images.githubusercontent.com/73607512/212018170-6c33a7e3-deaa-4f6e-9e59-3c43b0e7188a.PNG">